test/e2e/README.md

```diff
@@ -35,6 +35,22 @@ export GINKGO_FOCUS="upgrade"
 ./hack/ci-e2e.sh
 ```
 
+GINKGO_FOCUS can be set manually to run specific tests. The options for these can be found as the first string value of the line with "Describe" or "It". These can also be combined with other proceeding sections to match to even more specific test sections. The value GINKGO_FOCUS uses is a regexp that should match the description of the spec but not match the regexp specified in GINKGO_SKIP. Example:
```

Comment:
```suggestion
`GINKGO_FOCUS` can be set manually to run specific tests. The options for these can be found as the first string value of the line with `Describe` or `It`. These can also be combined with other proceeding sections to match to even more specific test sections. The value `GINKGO_FOCUS` uses is a regexp that should match the description of the spec but not match the regexp specified in `GINKGO_SKIP`. Example:
```

Use backticks appropriately where applicable.

Also, limit the line length to 80, the checker will fail otherwise when you come out of draft.